gradio/blocks.py

```diff
@@ -392,6 +392,8 @@ def postprocess_update_dict(
             )
         if postprocess:
             attr_dict["value"] = block.postprocess(update_dict["value"])
+            if isinstance(attr_dict["value"], (GradioModel, GradioRootModel)):
```

Comment:
Oh, so pydantic won't serialize it automatically?

Reply:
Yea it didn't work without me manually serializing the value when it was in an update dictionary. If it was a regular prediction, it would be fine.